sample-build-scripts/test/xamarin-uitest-android/appcenter-post-build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+echo "Found UI tests projects:"
+find $APPCENTER_SOURCE_DIRECTORY -regex '*.UITest.*\.csproj' -exec echo {} \;
+echo
+echo "Building UI test projects:"
+find $APPCENTER_SOURCE_DIRECTORY -name '*.UITest.csproj' -exec msbuild {} \;
+echo "Compiled projects to run UI tests:"
+find $APPCENTER_SOURCE_DIRECTORY -regex '*.bin.*UITest.*\.dll' -exec echo {} \;
+#echo "Running UI test in App Center Test:"
+echo "What is in the output directory"
+ls $APPCENTER_OUTPUT_DIRECTORY
+echo "Running test in App Center Test"
+APPPATH=$APPCENTER_OUTPUT_DIRECTORY/*.apk
```

Comment:
Add a check here to ensure that the various paths and directories have been correctly found - and bail out with an error if they haven't been found.